docs/wiki/ModulesDesign.md

```diff
@@ -414,17 +383,17 @@ Within a bicep file, follow the following conventions:
 
 ## Variables
 
-- camelCase, i.e `builtInRoleNames`
-- For modules that manage roleAssignments, update the list of roles to only be the applicable roles. You can find a helper script `Get-FormattedRBACRoles.ps1` in the `tools` folder of the repository.
+- Variable names are in camelCase, e.g. `builtInRoleNames`.
+- For modules that manage roleAssignments, typically cross-referenced `nested_rbac.bicep` in the `.bicep` folder, update the list of roles to only be the applicable roles. You can find a helper script `Get-FormattedRBACRoles.ps1` in the `utilities\tools\helper` folder of the repository.
```

Comment:
```suggestion
- Modules that manage roleAssignments typically have a cross-referenced `nested_rbac.bicep` file in their `.bicep` folder.  Those files require you to update their list of roles to only the applicable roles. You can find a helper script `Get-FormattedRBACRoles.ps1` in the `utilities/tools/helper` folder of the repository.
```

Reply:
I suggest moving this to the RBAC chapter on this article. Its not general consideration to variables, but this is more specific to RBAC.

Reply:
Moved and updated